src/model/utilities/randomNumbers.test.js

```diff
@@ -0,0 +1,24 @@
+import randomNumbers from './randomNumbers'
+
+describe('randomNumbers', ()=>{
+  describe('If we set lowest number as 1, highest number as 4 and length as 4',() => {
+    const result = randomNumbers(1, 4, 4);
+
+    test(`then we should get array with numbers 1, 2, 3, 4`, ()=> {
+      expect(result).toContain(1, 2, 3, 4);
+    });
+
+    test(`then we shouldn't get array with numbers 0, 5`, ()=> {
+      expect(result).not.toContain(5);
+    });
+
+    test(`then we should get array.length equal 4`, ()=> {
+      expect(result.length).toBe(4);
+    });
+
+    test(`then we should get different numbers 4`, ()=> {
+      const set = new Set(result);
+      expect(set.size).toBe(4);
+    });
```

Comment:
wytłumaczysz proszę o co chodzi z tym testem? nie do końca rozumiem

Reply:
Już tłumaczę. Chcę sprawdzić czy mam 4 różne liczby. W JS jest taki typ obiektu jak Set podobny do Array. Różni się tym, że w środku nie powtarzają się dane. I tutaj na podstawie tej swojej tablicy o nazwie result, tworzę właśnie Set. Że mam tam 1, 2, 3, 4 to Set ma wartości 1, 2, 3, 4 i jego wielkość (set.size) to 4. Ale gdybym np. stworzył ten set z liczb [1,1, 2, 2] to zawierałby liczbę 1 i 2 więc jego set.size było by równe 2.